test/index.spec.js

```diff
@@ -108,15 +108,21 @@ describe('pull-length-prefixed', () => {
         ).to.be.eql([Buffer.alloc(1, 0)])
```

Comment:
This is weird also, we've not pulled any values, yet the we've collected a value.

Reply:
Yeah, this should be empty. I believe it's an attempt to make
```js
pull(
  pull.values([]),
  lp.encode()
)
```

a shorthand of  
```js
pull(
  pull.values([Buffer.alloc(0)]),
  lp.encode()
)
```
which it really shouldn't be doing.